src/styles/blocks/container.scss

```diff
@@ -0,0 +1,5 @@
+.container {
```

Comment:
@huberciak yet centering calendar both vertically and horizontally is a must
please implement.